docs/deployment_patterns/how-to-instantiate-a-data-context-on-an-emr-spark-cluster.md

```diff
@@ -29,7 +29,7 @@ Steps
 
 2. **Configure a Data Context in code.**
 
-    Follow the steps for creating an in-code Data Context in [How to instantiate a Data Context without a yml file](../guides/setup/configuring-data-contexts/how-to-instantiate-a-data-context-without-a-yml-file.md).
+    Follow the steps for creating an in-code Data Context in [How to instantiate a Data Context without a yml file](../guides/setup/configuring_data_contexts/how-to-instantiate-a-data-context-without-a-yml-file.md).
```

Comment:
@cdkini GitHub still shows me file names containing hyphens, while the rest of the path uses the underscores.  Thanks.